src/components/wizard-internet/wizard-internet.js

```diff
@@ -8,17 +8,128 @@ export default module('app.components.wizard-internet', [])
     },
     // TODO: handle vpn
     controller: class WizardContactCtrl {
+
       constructor($scope) {
         'ngInject';
 
         this.newInternet = {};
+        this.internetTunnelEnabled = false;
+        this.meshTunnelEnabled = false;
+        this.internetTunnelConf = [
+          {
+            fileExtensions: '.ovpn,.cnf,text/*',
+            property: 'config',
+            tunnelType: 'internettunnel',
+            required: true,
+          },
+          {
+            fileExtensions: '.crt,text/*',
+            tunnelType: 'internettunnel',
+            property: 'cert',
+          },
+          {
+            fileExtensions: '.crt,text/*',
+            tunnelType: 'internettunnel',
+            property: 'cacert',
+          },
+          {
+            fileExtensions: '.key,text/*',
+            tunnelType: 'internettunnel',
+            property: 'key',
+          },
+          {
+            fileExtensions: '.key,text/*',
+            tunnelType: 'internettunnel',
+            property: 'takey',
```

Comment:
Ah, this is called `tls-auth`. We have to match the openvpn option names here, there is no additional translation of these options in the backend.